zipline/algorithm.py

```diff
@@ -1242,8 +1242,9 @@ def future_symbol(self, symbol):
 
     @api_method
     @preprocess(root_symbol=ensure_upper_case)
-    def future_chain(self, root_symbol, as_of_date=None):
-        """Look up a future chain with the specified parameters.
+    def future_chain(self, root_symbol, as_of_date=None, offset=0):
+        """
+        Look up a future chain with the specified parameters.
```

Comment:
`with the specified parameters` doesn't help the reader here.  "Look up a future chain." is a totally fine first sentence here.